dags/hacker_news.py

```diff
@@ -76,18 +63,66 @@ def process_stories_ids(date: str, **context) -> str:
         return stories_info
 
     @task()
-    def save_data():
+    def save_data(**context):
         """
         Save processed data to storage server
         """
+        import io
+
+        from minio import Minio     # type: ignore
+
+        global FILENAME
+        global BUCKET_PATH
+        global BUCKET_NAME
+
         logger.info('Save stories data to minio')
-        pass
+
+        logger.info('Load data from xCom')
+        ti = context['ti']
+        stories_info = ti.xcom_pull(task_ids='process_stories_ids')
+        logger.info(os.environ.get('MINIO_SECURE'))
+
+        client = Minio(
```

Comment:
Have you read about hooks in airflow? What do you think about using built-in s3hook as a client for minio? What is the advantages/disadvantages of each approach?

Reply:
I guess that we wouldn't require additional third-party requirement (minio client package) in case when we using built-in s3hook.

Reply:
I think that even the interest I should try to use built-in s3hook